_ml-commons-plugin/algorithms.md

```diff
@@ -198,6 +198,93 @@ time_zone | string | The time zone for the time_field field | "UTC"
 
 For FIT RCF, you can train the model with historical data and store the trained model in your index. The model will be deserialized and predict new data points when using the Predict API. However, the model in the index will not be refreshed with new data, because the model is fixed in time.
 
+## RCFSummarize
+
+RCFSummarize is a clustering algorithm based on the Clustering Using REpresentatives (CURE) algorithm. Compared to [k-means](#k-means), which uses random iterations to cluster, RCFSummarize uses a hierarchical-based clustering technique. The algorithm starts, with a set of randomly selected centroids larger than the centroid’s ground-truth distribution. During iteration, centroid pairs too close to each other automatically merge. Therefore, the number of centroids (`max_k`) converge to a rational number of clusters that fits ground-truth, as opposed to a fixed `k` number of clusters.  
```

Comment:
Instead of "hierarchical-based clustering", would just "hierarchical clustering" make sense? Remove the comma in the third sentence after "starts". If "centroid's" refers to multiple centroids, it should be "centroids'" (plural possessive). I'm not seeing "ground truth" routinely hyphenated in other sources, so let's remove the hyphen in both instances here.